web/src/app/common/publicRoute/PublicRoute.js

```diff
@@ -0,0 +1,21 @@
+import { Redirect, Route } from 'react-router-dom';
```

Comment:
There looks like a lot of code duplication between PublicRoute and PrivateRoute. Can we combine these somehow? Perhaps pass a flag in to indicate if the route should be private/public?

Reply:
I like it. I'll change it to something like "ProtectedRoute" with a prop.